packages/openbox.rb

```diff
@@ -0,0 +1,83 @@
+require 'package'
+
+class Openbox < Package
+  description 'Openbox is a highly configurable, next generation window manager with extensive standards support.'
+  homepage 'http://openbox.org/'
+  version '3.6.1'
+  compatibility 'all'
+  source_url 'http://openbox.org/dist/openbox/openbox-3.6.1.tar.xz'
+  source_sha256 'abe75855cc5616554ffd47134ad15291fe37ebbebf1a80b69cbde9d670f0e26d'
+
+  depends_on 'libev'
+  depends_on 'startup_notification'
+  depends_on 'xcb_util_cursor'
+  depends_on 'xcb_util_keysyms'
+  depends_on 'xcb_util_wm'
+  depends_on 'xcb_util_xrm'
+  depends_on 'yajl'
+  depends_on 'wayland' => :build
+  depends_on 'xorg_proto'
+  depends_on 'glib'
+  depends_on 'libxfixes'
+  depends_on 'libsm'
+  depends_on 'pango' => :build
+  depends_on 'libx11' => :build
+  #depends_on 'adwaita_icon_theme'
+  depends_on 'xdpyinfo'
+  depends_on 'sommelier'
+
+  ENV['CFLAGS'] = "-lX11 -lXau"
+  def self.build
+    system "./configure #{CREW_OPTIONS}"
+    system "make"
+  end
+
+  def self.install
+    system "make", "DESTDIR=#{CREW_DEST_DIR}", "install"
+    system "echo '#!/bin/bash' > startopenbox"
+    system "cat <<'EOF'> startopenbox
```

Comment:
You could move the `#!/bin/bash` line here instead.

Reply:
Done